### PR TITLE
add support for markdown format

### PR DESCRIPTION
**Pull request by kalidiagne**
This PR adds support for Markdown on the project details page. (screens attached to showcase all the different styling)
Also, now that we support markdown, every project is not split into a separate file to make  `projects.ts` more manageable. 

##### Example 1

Screenshot 2023-08-09 at 16 09 44

##### Example 2

Screenshot 2023-08-09 at 16 10 01

#### Example 3

Screenshot 2023-08-10 at 01 05 11

### CODE REVIEW

File: components/ui/markdown.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  React.createElement(tag, {
+    ...props,
+  })
+
+// Styling for HTML attributes for markdown component
+const REACT_MARKDOWN_CONFIG: Components = {
+  a: ({ node, ...props }) =>
+    createMarkdownElement("a", {
+      className: "text-orange",
+      ...props,
```
Comment (kalidiagne) [inline components/ui/markdown.tsx:14]: @thebeyondr @ChialiT  do we need to open links in a different tab?

File: components/ui/markdown.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    }),
```
Comment (kalidiagne) [inline components/ui/markdown.tsx:15]: @thebeyondr @ChialiT  do we need to open links in a different tab?

Comment (kalidiagne) [reply components/ui/markdown.tsx]: awesome, added!

**Review by thebeyondr — APPROVED** [review]
🚀

**Review by ChialiT — APPROVED** [review]

**Comment by ChialiT** [comment]
LGTM, lets merge it~

**Pull request merged**